tools/macos_buildenv.sh

```diff
@@ -56,22 +110,14 @@ case "$1" in
         mkdir -p "${BUILDENV_BASEPATH}"
         if [ ! -d "${BUILDENV_PATH}" ]; then
             if [ "$1" != "--profile" ]; then
-                echo "Build environment $BUILDENV_NAME not found in mixxx repository, downloading https://downloads.mixxx.org/dependencies/${BUILDENV_BRANCH}/macOS/${BUILDENV_NAME}.zip"
-                http_code=$(curl -sI -w "%{http_code}" "https://downloads.mixxx.org/dependencies/${BUILDENV_BRANCH}/macOS/${BUILDENV_NAME}.zip" -o /dev/null)
-                if [ "$http_code" -ne 200 ]; then
-                    echo "Downloading  failed with HTTP status code: $http_code"
-                    exit 1
-                fi
-                curl "https://downloads.mixxx.org/dependencies/${BUILDENV_BRANCH}/macOS/${BUILDENV_NAME}.zip" -o "${BUILDENV_PATH}.zip"
-                OBSERVED_SHA256=$(shasum -a 256 "${BUILDENV_PATH}.zip"|cut -f 1 -d' ')
-                if [[ "$OBSERVED_SHA256" == "$BUILDENV_SHA256" ]]; then
-                    echo "Download matched expected SHA256 sum $BUILDENV_SHA256"
+                if [ "$BUILDENV_SRC" = "gh" ]; then
+                    echo "Build environment $BUILDENV_NAME not found in mixxx repository, downloading from Mixxx's VCPKG repo"
```

Comment:
```suggestion
                    echo "Build environment $BUILDENV_NAME not found at ${BUILDENV_BASEPATH}, downloading from https://github.com/mixxxdj/vcpkg/releases"
```
Showing the URLs for a bit more confidence.